src/DynamicObj/DynamicObj.fs

```diff
@@ -137,6 +164,23 @@ type DynamicObj() =
         #endif
         |> Seq.filter (fun kv -> kv.Key.ToLower() <> "properties")
 
+    /// Copies all dynamic members of the DynamicObj to the target DynamicObj.
+    member this.CopyDynamicPropertiesTo(target:#DynamicObj, ?overWrite) =
+        let overWrite = Option.defaultValue false overWrite
```

Comment:
There is also `defaultArg`, i vaguely remember that being better to use in these scenarios, but i do not remember why. Your call

Reply:
For Fable you mean?

Reply:
no i mean in general for determining default args from oprional arguments

Reply:
I think here it doesn't matter. 

In cases where the default expression is computationally expensive, it's worth to switch to an explicit match. But not sure whether `Option.defaultValue` and `defaultArg` even have different implementations.